### PR TITLE
added is_secret field to SMSTemplate

### PR DESCRIPTION
**Pull request by jsilhan**

### CODE REVIEW

File: pymess/locale/cs/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -564,19 +564,23 @@ msgstr "vztažený objekt SMS zprávy"
 msgid "related objects of SMS messages"
 msgstr "vztažené objekty SMS zpráv"
 
-#: models/sms.py:68
+#: models/sms.py:62
+msgid "is secret"
+msgstr "tajný?"
```
Comment (jsilhan) [inline pymess/locale/cs/LC_MESSAGES/django.po]: translation is the same as in core project

**Comment by jsilhan** [comment]
@dstlmrk please review

File: pymess/locale/cs/LC_MESSAGES/django.po (hunk 1 above)
Comment (dstlmrk) [reply pymess/locale/cs/LC_MESSAGES/django.po]: please without question mark

**Review by dstlmrk — COMMENTED** [review]
it seems good

File: pymess/locale/cs/LC_MESSAGES/django.po (hunk 1 above)
Comment (jsilhan) [reply pymess/locale/cs/LC_MESSAGES/django.po]: changed.

**Pull request merged**